packages/extension/package.json

```diff
@@ -27,7 +27,7 @@
     "@typescript-eslint/eslint-plugin": "^5.10.1",
     "@typescript-eslint/parser": "^5.10.1",
     "@vitejs/plugin-react": "^3.0.0",
-    "bip39": "^3.1.0",
+    "bip39": "3.0.4",
```

Comment:
Question: why downgrade?

Reply:
This creates two versions because we use fixed version (`3.0.4` for bip39 ) for all crypto stuff in web3 SDK to protect from supply-chain attack.